### ISSUE TITLE
Feature Request: Option to rename workers remote control command exchange

### ISSUE THREAD

**Issue by art-vasilyev** [celery/celery#5189]
Sometimes several apps with celery are running on the same server. As for now, when using RabbitMQ as the broker, recommended way to isolate different celery apps is to create separate RabbitMQ vhost for each application.

It would be good to add a configuration option to rename default exchanges (celery.pidbox, celeryev, reply.celery.pidbox) to have ability to run several celery apps on the same RabbitMQ vhost. Task exchanges and queues already have such options (task_default_exchange, task_default_queue).
'celery.pidbox' is hardcoded in https://github.com/celery/celery/blob/master/celery/app/control.py#L163,
'celeryev' in https://github.com/celery/celery/blob/master/celery/events/event.py#L16

**Comment by auvipy** [comment]
what is your proposed solution?

**Comment by art-vasilyev** [comment]
Add option like `event_default_exchange` ("celeryev" by default) and `mailbox_default_exchange` ("celery" by default). Mailbox initialization (https://github.com/celery/celery/blob/master/celery/app/control.py#L163) would look like:
```
self.mailbox = self.Mailbox(
    app.conf.mailbox_default_exchange,
    type='fanout',
    accept=['json'],
    producer_pool=lazy(lambda: self.app.amqp.producer_pool),
    queue_ttl=app.conf.control_queue_ttl,
    reply_queue_ttl=app.conf.control_queue_ttl,
    queue_expires=app.conf.control_queue_expires,
    reply_queue_expires=app.conf.control_queue_expires,
)
```
As event exchange initializes on module import, it may require lazy initialization to get exchange name from the app config. 

If this idea doesn't contradict the project, I may try to implement it and submit a pull request.

**Comment by art-vasilyev** [comment]
Resolved in #5195
